test/integration/compile-test.coffee

```diff
@@ -38,163 +38,168 @@ describe('compile() · all API description formats', ->
 
   describe('causing an error in the parser', ->
     fixtures.parserError.forEachDescribe(({source}) ->
-      errors = undefined
-      transactions = undefined
+      compilationResult = undefined
 
       beforeEach((done) ->
-        compileFixture(source, (err, compilationResult) ->
-          return done(err) if err
-          {errors, transactions} = compilationResult
-          done()
+        compileFixture(source, (args...) ->
+          [err, compilationResult] = args
+          done(err)
         )
       )
 
       it('is compiled into zero transactions', ->
-        assert.equal(transactions.length, 0)
+        assert.deepEqual(compilationResult.transactions, [])
```

Comment:
Because when this fails, I can see the extra transaction in the test report. Otherwise I'd just see `3 ≠ 0`, which isn't very helpful and requires me to go to the test and add `console.log`s to see what exactly is wrong.